Automattic/wp-calypso
# Update copy for Scan Daily, Anti-spam, and Search product cards

### Changes proposed in this Pull Request

This PR includes the final copy for the Jetpack Scan _Daily_, Jetpack Anti-spam, and Jetpack Search product cards.

Jetpack Complete was handled in https://github.com/Automattic/wp-calypso/pull/45009.
Security in https://github.com/Automattic/wp-calypso/pull/45035.
Backups in https://github.com/Automattic/wp-calypso/pull/45042.

### Implementation notes

- We'll use Jetpack Scan _Daily_ instead of Jetpack Scan
- Made sure descriptions end by the correct punctuation
- Made sure Anti-spam is spelled correctly

### Testing instructions

- Visit the _Plans_ page with the offer reset flow enabled
- Show all products and plans
- Check the copy of the product cards against the copydeck

### Screenshots

Jetpack Scan _Daily_
<img width="439" alt="Screen Shot 2020-08-20 at 11 43 32 AM" src="https://user-images.githubusercontent.com/1620183/90794212-85936a00-e2da-11ea-8d9e-e8b506189a56.png">

Jetpack Anti-spam
<img width="448" alt="Screen Shot 2020-08-20 at 11 43 38 AM" src="https://user-images.githubusercontent.com/1620183/90794220-89bf8780-e2da-11ea-8cb0-1c61966465c6.png">

Jetpack Search
<img width="458" alt="Screen Shot 2020-08-20 at 11 43 43 AM" src="https://user-images.githubusercontent.com/1620183/90794237-8deba500-e2da-11ea-80e0-d2c0eac01cb9.png">
